test/index.js

```diff
@@ -0,0 +1,15 @@
+import assert      from 'assert'
+import superagent  from 'superagent'
+import startServer from '../lib/server'
+
+startServer(8001)
```

Comment:
What if you started the server without a port? You'd never have to worry about conflicting connections.